l5kit/l5kit/data/map_api.py

```diff
@@ -174,7 +177,8 @@ def get_crosswalk_coords(self, element_id: str) -> dict:
             traffic_element.geo_frame,
         )
 
-        return {"xyz": xyz}
+        xyz[:, -1] = 1.0
+        return {"xyz": xyz.T}
```

Comment:
I can see the point of caching this to avoid stack at runtime, but we may still want to include either the two lanes or at least the length of the first, so that we can always unpack the two apart (I'm thinking about centre line support right now)

Reply:
The cheapest/easiest solution if you add the length of the first line. I think `map_api` should handle the centerline calculation as well (cachable).

Reply:
yeah, I agree on that and I'm fine with having the length included

Reply:
One note: I changed the returned `xyz` format to `xyz.T`.